dockerfiles/init/modules/openshift/files/scripts/ocp.sh

```diff
@@ -219,6 +303,27 @@ parse_args() {
       DEPLOY_SCRIPT_ARGS="-c rollupdate"
     fi
 
+    if [[ "$@" == *"--deploy-che"* ]]; then
+      CHE_DEPLOY=true
+    fi
+
+    if [[ "$@" == *"--remove-che"* ]]; then
+      CHE_REMOVE_PROJECT=true
+    fi
+
+    if [[ "$@" == *"--replace-stacks"* ]]; then
+      CHE_REPLACE_STACKS=true
+    fi
+    for i in "${@}"
```

Comment:
This doesn't look consistent with other places where arguments are parsed. I believe that 3 ways of parsing arguments is a thing that we should avoid.

Reply:
Not sure I understand the comment. I add a separate for loop to catch --help if the user does something like `--replace-stacks --help`. However I can put --help back into the other for loop as this case is rare. Let me know.

Reply:
To parse arguments we use if clauses (lines 298-316), loop for checking help argument, loop for starting commends. This is too complex for such a small and simple thing as parsing arguments.
-1 to add that help loop. I'd rather refactor this to use a simpler approach such as parsing arguments in a loop, starting commands afterward.

Reply:
Will do.